client/battle/BattleWindow.cpp

```diff
@@ -407,7 +407,7 @@ void BattleWindow::bFleef()
 		auto txt = boost::format(CGI->generaltexth->allTexts[340]) % heroName; //The Shackles of War are present.  %s can not retreat!
 
 		//printing message
-		owner.curInt->showInfoDialog(boost::to_string(txt), comps);
+		owner.curInt->showInfoDialog(boost::str(txt), comps);

```

Comment:
std actually has to_string unlike str

Reply:
However std does not have anything like boost::format.

What happened here:
- I removed "including everything" include for boost thread and only included stuff we actually use
- because of this change, we no longer include boost::to_string (but still include boost::format)
- so I changes boost::to_string to boost::str which is standard way to convert boost::format to string

Reply:
Yeah, in 17 we have only snprintf, in 20 there's std::format